src/utils/createWhereObj.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const { Op } = require('sequelize');
+const createWhereObj = (params) => {
+  const obj = { where: {} };
+  const { userId, categories, from, to } = params;
+
+  if (userId) {
+    obj.where.userId = +userId;
```

Comment:
There is no need to convert userId to number, it should be validated in the controller.